engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -2464,6 +2465,20 @@ namespace dmGameObject
 
         bool ret = true;
 
+
+        UpdateContext dynamic_update_context;
+        dynamic_update_context = *update_context;
+
+        // pass it as unscaled time
+        if (update_context->m_TimeScale > 0.001f )
+        {
+            dynamic_update_context.m_AccumFrameTime = collection->m_FixedAccumTime / update_context->m_TimeScale;
```

Comment:
The collection has time scaled accumulated time, and we need to pass unscaled time down through the hierarchy (to other collection proxies)